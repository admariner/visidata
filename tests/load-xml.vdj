#!vd -p
{"sheet": null, "col": null, "row": null, "longname": "open-file", "input": "sample_data/benchmark.xml", "keystrokes": "o", "comment": null}
