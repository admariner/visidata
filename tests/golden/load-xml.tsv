tag	children	text
data	51	
  
row	8	
    
index	0	0
Date	0	7/3/2018 1:47p
Customer	0	Robert Armstrong
SKU	0	FOOD213
Item	0	BFF Oh My Gravy! Beef & Salmon 2.8oz
Quantity	0	4
Unit	0	$12.95
Paid	0	$51.8
row	8	
    
index	0	1
Date	0	7/3/2018 3:32p
Customer	0	Kyle Kennedy
SKU	0	FOOD121
Item	0	Food, Adult Cat - 3.5 oz
Quantity	0	1
Unit	0	$4.22
Paid	0	$4.22
row	8	
    
index	0	2
Date	0	7/5/2018 4:15p
Customer	0	Douglas "Dougie" Powers
SKU	0	FOOD121
Item	0	Food, Adult Cat 3.5 oz
Quantity	0	1
Unit	0	$4.22
Paid	0	$4.22
row	8	
    
index	0	3
Date	0	7/6/2018 12:15p
Customer	0	桜 高橋 (Sakura Takahashi)
SKU	0	FOOD122
Item	0	Food, Senior Wet Cat - 3 oz
Quantity	0	12
Unit	0	$1.29
Paid	0	157¥
row	8	
    
index	0	4
Date	0	7/10/2018 10:28a
Customer	0	David Attenborough
SKU	0	NSCT201
Item	0	Food, Salamander
Quantity	0	30
Unit	0	$.05
Paid	0	$1.5
row	8	
    
index	0	5
Date	0	7/10/2018 5:23p
Customer	0	Susan Ashworth
SKU	0	CAT060
Item	0	Cat, Korat (Felis catus)
Quantity	0	1
Unit	0	$720.42
Paid	0	$720.42
row	8	
    
index	0	6
Date	0	7/10/2018 5:23p
Customer	0	Susan Ashworth
SKU	0	FOOD130
Item	0	Food, Kitten 3kg
Quantity	0	1
Unit	0	$14.94
Paid	0	$14.94
row	8	
    
index	0	7
Date	0	7/13/2018 10:26a
Customer	0	Wil Wheaton
SKU	0	NSCT523
Item	0	Monster, Rust (Monstrus gygaxus)
Quantity	0	1
Unit	0	$39.95
Paid	0	$39.95
row	8	
    
index	0	8
Date	0	7/13/2018 3:49p
Customer	0	Robert Armstrong
SKU	0	FOOD216
Item	0	BFF Oh My Gravy! Chicken & Shrimp 2.8oz
Quantity	0	4
Unit	0	$12.95
Paid	0	$51.8
row	8	
    
index	0	9
Date	0	7/17/2018 9:01a
Customer	0	Robert Armstrong
SKU	0	FOOD217
Item	0	BFF Oh My Gravy! Duck & Tuna 2.8oz
Quantity	0	4
Unit	0	$12.95
Paid	0	$51.8
row	8	
    
index	0	10
Date	0	7/17/2018 11:30a
Customer	0	Helen Halestorm
SKU	0	LAGO342
Item	0	Rabbit (Oryctolagus cuniculus)
Quantity	0	2
Unit	0	$32.94
Paid	0	$65.88
row	8	
    
index	0	11
Date	0	7/18/2018 12:16p
Customer	0	桜 高橋 (Sakura Takahashi)
SKU	0	FOOD122
Item	0	Food, Senior Wet Cat - 3 oz
Quantity	0	6
Unit	0	$1.29
Paid	0	157¥
row	8	
    
index	0	12
Date	0	7/19/2018 10:28a
Customer	0	Rubeus Hagrid
SKU	0	FOOD170
Item	0	Food, Dog - 5kg
Quantity	0	5
Unit	0	$44.95
Paid	0	$224.75
row	8	
    
index	0	13
Date	0	7/20/2018 2:13p
Customer	0	Jon Arbuckle
SKU	0	FOOD167
Item	0	Food, Premium Wet Cat - 3.5 oz
Quantity	0	50
Unit	0	$3.95
Paid	0	$197.5
row	8	
    
index	0	14
Date	0	7/23/2018 1:41p
Customer	0	Robert Armstrong
SKU	0	FOOD215
Item	0	BFF Oh My Gravy! Lamb & Tuna 2.8oz
Quantity	0	4
Unit	0	$12.95
Paid	0	$51.8
row	8	
    
index	0	15
Date	0	7/23/2018 4:23p
Customer	0	Douglas "Dougie" Powers
SKU	0	TOY235
Item	0	Laser Pointer
Quantity	0	1
Unit	0	$16.12
Paid	0	$16.12
row	8	
    
index	0	16
Date	0	7/24/2018 12:16p
Customer	0	桜 高橋 (Sakura Takahashi)
SKU	0	FOOD122
Item	0	Food, Senior Wet Cat - 3 oz
Quantity	0	3
Unit	0	$1.29
Paid	0	157¥
row	8	
    
index	0	17
Date	0	7/26/2018 4:39p
Customer	0	Douglas "Dougie" Powers
SKU	0	FOOD420
Item	0	Food, Shark - 10 kg
Quantity	0	1
Unit	0	$15.70
Paid	0	$15.7
row	8	
    
index	0	18
Date	0	7/27/2018 12:16p
Customer	0	桜 高橋 (Sakura Takahashi)
SKU	0	FOOD122
Item	0	Food, Senior Wet Cat - 3 oz
Quantity	0	3
Unit	0	$1.29
Paid	0	157¥
row	8	
    
index	0	19
Date	0	7/30/2018 12:17p
Customer	0	桜 高橋 (Sakura Takahashi)
SKU	0	RETURN
Item	0	Food, Senior Wet Cat - 3 oz
Quantity	0	1
Unit	0	$1.29
Paid	0	157¥
row	8	
    
index	0	20
Date	0	7/31/2018 5:42p
Customer	0	Rubeus Hagrid
SKU	0	CAT060
Item	0	Food, Dragon - 50kg
Quantity	0	5
Unit	0	$720.42
Paid	0	$3602.1
row	8	
    
index	0	21
Date	0	8/1/2018 2:44p
Customer	0	David Attenborough
SKU	0	FOOD360
Item	0	Food, Rhinocerous - 50kg
Quantity	0	4
Unit	0	$5.72
Paid	0	$22.88
row	8	
    
index	0	22
Date	0	8/2/2018 5:12p
Customer	0	Susan Ashworth
SKU	0	CAT110
Item	0	Cat, Maine Coon (Felix catus)
Quantity	0	1
Unit	0	$1,309.68
Paid	0	$1309.68
row	8	
    
index	0	23
Date	0	8/2/2018 5:12p
Customer	0	Susan Ashworth
SKU	0	FOOD130
Item	0	Food, Kitten 3kg
Quantity	0	3
Unit	0	$14.94
Paid	0	$44.82
row	8	
    
index	0	24
Date	0	8/6/2018 10:21a
Customer	0	Robert Armstrong
SKU	0	FOOD212
Item	0	BFF Oh My Gravy! Beef & Chicken 2.8oz
Quantity	0	4
Unit	0	$12.95
Paid	0	$51.8
row	8	
    
index	0	25
Date	0	8/7/2018 4:12p
Customer	0	Juan Johnson
SKU	0	REPT082
Item	0	Kingsnake, California (Lampropeltis getula)
Quantity	0	1
Unit	0	$89.95
Paid	0	$89.95
row	8	
    
index	0	26
Date	0	8/7/2018 4:12p
Customer	0	Juan Johnson
SKU	0	RDNT443
Item	0	Mouse, Pinky (Mus musculus)
Quantity	0	1
Unit	0	$1.49
Paid	0	$1.49
row	8	
    
index	0	27
Date	0	8/10/2018 4:31p
Customer	0	Robert Armstrong
SKU	0	FOOD211
Item	0	BFF Oh My Gravy! Chicken & Turkey 2.8oz
Quantity	0	4
Unit	0	$12.95
Paid	0	$51.8
row	8	
    
index	0	28
Date	0	8/13/2018 2:07p
Customer	0	Monica Johnson
SKU	0	RDNT443
Item	0	Mouse, Pinky (Mus musculus)
Quantity	0	1
Unit	0	$1.49
Paid	0	$1.49
row	8	
    
index	0	29
Date	0	8/13/2018 2:08p
Customer	0	María Fernández
SKU	0	FOOD146
Item	0	Forti Diet Prohealth Mouse/Rat 3lbs
Quantity	0	2
Unit	0	$2.00
Paid	0	$4.0
row	8	
    
index	0	30
Date	0	8/15/2018 11:57a
Customer	0	Mr. Praline
SKU	0	RETURN
Item	0	Parrot, Norwegian Blue (Mopsitta tanta)
Quantity	0	1
Unit	0	$2300.00
Paid	0	-$2300.0
row	8	
    
index	0	31
Date	0	8/15/2018 3:48p
Customer	0	Kyle Kennedy
SKU	0	FOOD121
Item	0	Food, Adult Cat - 3.5 oz
Quantity	0	2
Unit	0	$4.22
Paid	0	$8.44
row	8	
    
index	0	32
Date	0	8/16/2018 11:50a
Customer	0	Helen Halestorm
SKU	0	RETURN
Item	0	Rabbit (Oryctolagus cuniculus)
Quantity	0	6
Unit	0	$0
Paid	0	$0.0
row	8	
    
index	0	33
Date	0	8/16/2018 4:00p
Customer	0	Kyle Kennedy
SKU	0	DOG010
Item	0	Dog, Golden Retriever (Canis lupus familiaris)
Quantity	0	1
Unit	0	$2,495.99
Paid	0	$2495.99
row	8	
    
index	0	34
Date	0	8/16/2018 5:15p
Customer	0	Michael Smith
SKU	0	BIRD160
Item	0	Parakeet, Blue (Melopsittacus undulatus)
Quantity	0	1
Unit	0	29.95
Paid	0	$31.85
row	8	
    
index	0	35
Date	0	8/17/2018 9:26a
Customer	0	Rubeus Hagrid
SKU	0	NSCT201
Item	0	Food, Spider
Quantity	0	5
Unit	0	$.05
Paid	0	$0.25
row	8	
    
index	0	36
Date	0	8/20/2018 9:36a
Customer	0	Kyle Kennedy
SKU	0	RETURN
Item	0	Dog, Golden Retriever (Canis lupus familiaris)
Quantity	0	1
Unit	0	$1,247.99
Paid	0	-$1247.99
row	8	
    
index	0	37
Date	0	8/20/2018 1:47p
Customer	0	מרוסיה ניסנהולץ אבולעפיה
SKU	0	GOAT224
Item	0	Goat, American Pygmy (Capra hircus)
Quantity	0	1
Unit	0	₪499
Paid	0	$160.51
row	8	
    
index	0	38
Date	0	8/20/2018 3:31p
Customer	0	Monica Johnson
SKU	0	NSCT201
Item	0	Crickets, Adult Live (Gryllus assimilis)
Quantity	0	30
Unit	0	$.05
Paid	0	$1.5
row	8	
    
index	0	39
Date	0	8/20/2018 5:12p
Customer	0	David Attenborough
SKU	0	NSCT084
Item	0	Food, Pangolin
Quantity	0	30
Unit	0	$.17
Paid	0	$5.10
row	8	
    
index	0	40
Date	0	8/21/2018 12:13p
Customer	0	Robert Armstrong
SKU	0	FOOD214
Item	0	BFF Oh My Gravy! Duck & Salmon 2.8oz
Quantity	0	4
Unit	0	$12.95
Paid	0	$51.8
row	8	
    
index	0	41
Date	0	8/22/2018 9:38a
Customer	0	David Attenborough
SKU	0	BIRD160
Item	0	Food, Quoll
Quantity	0	1
Unit	0	29.95
Paid	0	$29.95
row	8	
    
index	0	42
Date	0	8/22/2018 2:13p
Customer	0	Jon Arbuckle
SKU	0	FOOD170
Item	0	Food, Adult Dog - 5kg
Quantity	0	1
Unit	0	$44.95
Paid	0	$44.95
row	8	
    
index	0	43
Date	0	8/22/2018 5:49p
Customer	0	מרוסיה ניסנהולץ
SKU	0	SFTY052
Item	0	Fire Extinguisher, kitchen-rated
Quantity	0	1
Unit	0	$61.70
Paid	0	$61.70
row	8	
    
index	0	44
Date	0	8/24/2018 11:42a
Customer	0	Robert Armstrong
SKU	0	FOOD218
Item	0	BFF Oh My Gravy! Chicken & Salmon 2.8oz
Quantity	0	4
Unit	0	$12.95
Paid	0	$51.8
row	8	
    
index	0	45
Date	0	8/27/2018 3:05p
Customer	0	Monica Johnson
SKU	0	NSCT443
Item	0	Mealworms, Large (Tenebrio molitor) 100ct
Quantity	0	1
Unit	0	$1.99
Paid	0	$1.99
row	8	
    
index	0	46
Date	0	8/28/2018 5:32p
Customer	0	Susan Ashworth
SKU	0	CAT020
Item	0	Cat, Scottish Fold (Felis catus)
Quantity	0	1
Unit	0	$1,964.53
Paid	0	$1964.53
row	8	
    
index	0	47
Date	0	8/28/2018 5:32p
Customer	0	Susan Ashworth
SKU	0	FOOD130
Item	0	Food, Kitten 3kg
Quantity	0	2
Unit	0	$14.94
Paid	0	$29.88
row	8	
    
index	0	48
Date	0	8/29/2018 10:07a
Customer	0	Robert Armstrong
SKU	0	FOOD219
Item	0	BFF Oh My Gravy! Chicken & Pumpkin 2.8oz
Quantity	0	4
Unit	0	$12.95
Paid	0	$51.8
row	8	
    
index	0	49
Date	0	8/31/2018 12:00a
Customer	0	Robert Armstrong
SKU	0	FOOD219
Item	0	BFF Oh My Gravy! Chicken & Pumpkin 2.8oz
Quantity	0	144
Unit	0	$12.95
Paid	0	$1864.8
row	8	
    
index	0	50
Date	0	8/31/2018 5:57p
Customer	0	Juan Johnson
SKU	0	REPT217
Item	0	Lizard, Spinytail (Uromastyx ornatus)
Quantity	0	1
Unit	0	$99.95
Paid	0	$99.95
